appengine/app.yaml

```diff
@@ -1,7 +1,5 @@
-application: cayley-test
-version: 1
 runtime: go
-api_version: go1.6.3
+api_version: go1
```

Comment:
appengine only supports go1 in this field. see here: https://cloud.google.com/appengine/docs/go/config/appref